extensions/amp-story/1.0/localization.js

```diff
@@ -95,6 +94,9 @@ const LANGUAGE_CODE_CHUNK_REGEX = /\w+/gi;
  * @return {!Array<string>} A list of language codes.
  */
 export function getLanguageCodesFromString(languageCode) {
+  if (!languageCode) {
+    return ['en'];
```

Comment:
Now that ``en`` is the default language, what do you think if renaming ``DEFAULT_LANGUAGE_CODE`` into ``FALLBACK_LANGUAGE_CODE``?